nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -29,72 +29,100 @@
 - All [ROCm](https://rocm.docs.amd.com/en/latest/) packages have been updated to 5.7.0.
   - [ROCm](https://rocm.docs.amd.com/en/latest/) package attribute sets are versioned: `rocmPackages` -> `rocmPackages_5`.
 
-## New Services {#sec-release-23.11-new-services}
+## New Modules {#sec-release-23.11-new-modules}
 
-- [MCHPRS](https://github.com/MCHPR/MCHPRS), a multithreaded Minecraft server built for redstone. Available as [services.mchprs](#opt-services.mchprs.enable).
+<!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
-- [acme-dns](https://github.com/joohoi/acme-dns), a limited DNS server to handle ACME DNS challenges easily and securely. Available as [services.acme-dns](#opt-services.acme-dns.enable).
+[hardware.infiniband.enable](#opt-hardware.infiniband.enable)
```

Comment:
I suppose this is sorted by alphabetic order on the configuration fragment?

Reply:
Yes, that is the intention, since they are all thrown at once to readers.